integration/data/test_restore_inc.py

```diff
@@ -137,8 +137,8 @@ def restore_inc_test(grpc_controller,  # NOQA
 
     assert not path.exists(STANDBY_REPLICA1_PATH + delta_file1)
     assert not path.exists(STANDBY_REPLICA2_PATH + delta_file1)
-    volume_info = cmd.info(CONTROLLER_NO_FRONTEND)
-    assert volume_info['lastRestored'] == backup1_name
+    volume_info = cmd.restore_status(CONTROLLER_NO_FRONTEND)
```

Comment:
it's no longer `volume_info`.

Also, need to check all the replicas instead of just the first one.